### PR TITLE
node docker update

### PR DESCRIPTION
**Pull request by Goodsmileduck**

### CODE REVIEW

File: Dockerfile-mysterium-node [hunk 1]
```diff
@@ -1,15 +1,21 @@
...
-ARG PACKAGE_FILE
-COPY $PACKAGE_FILE /tmp/mysterium-node.deb
+ENV VERSION 0.0.6
+ENV NODE 321
 
 # Install packages
 RUN apt-get update \
-    && dpkg --install /tmp/mysterium-node.deb
-    && apt-get install --fix-broken \
+    && apt install -y wget openvpn iptables\
```
Comment (Waldz) [inline Dockerfile-mysterium-node]: I would remove: "openvpn iptables", they are required in .deb package with more exact version

File: Dockerfile-mysterium-node [hunk 2]
```diff
@@ -1,15 +1,21 @@
 FROM ubuntu:latest
 MAINTAINER Valdas Petrulis <petrulis.valdas@gmail.com>
 
-# e.g. PACKAGE_FILE="build/package/mysterium-node_0.0.1_linux_amd64.deb"
-ARG PACKAGE_FILE
-COPY $PACKAGE_FILE /tmp/mysterium-node.deb
+ENV VERSION 0.0.6
+ENV NODE 321
```
Comment (Waldz) [inline Dockerfile-mysterium-node]: Not good hardcoding NODE key, it should be injected with container environment variables

File: entrypoint-node.sh [hunk 3]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -e
+
+mkdir -p /dev/net
```
Comment (Waldz) [inline entrypoint-node.sh]: Can this be done In Dockerfile, not during startup?

File: Dockerfile-mysterium-node [hunk 4]
```diff
@@ -1,15 +1,21 @@
...
+    && rm mysterium-node_linux_amd64.deb
 
-ENTRYPOINT ["/usr/bin/mysterium_server"]
\ No newline at end of file
+COPY entrypoint-node.sh /entrypoint-node.sh
+RUN chmod +x /entrypoint-node.sh
+
+ENTRYPOINT ["/entrypoint-node.sh"]
+
+CMD mysterium_server --config-dir=/etc/mysterium-node --node=${NODE} --runtime-dir=/tmp
```
Comment (Waldz) [inline Dockerfile-mysterium-node]: Exact starting command should be like this:
```
sudo /usr/bin/mysterium_server --node=NODE_KEY --config-dir=/etc/mysterium-node --runtime-dir=/var/run/mysterium-node
```

File: Dockerfile-mysterium-node [hunk 5]
```diff
@@ -1,15 +1,21 @@
 FROM ubuntu:latest
```
Comment (ewrogers) [inline Dockerfile-mysterium-node]: `FROM debian:jessie-slim` yields a smaller container size.

File: entrypoint-node.sh (hunk 3 above)
Comment (ewrogers) [inline entrypoint-node.sh]: Yes, seems you can add
```
# Create OpenVPN tunnel device
RUN set -e \
    && mkdir -p /dev/net \
    && mknod /dev/net/tun c 10 200 \
    && chmod 600 /dev/net/tun
```

File: docker-compose.yml [hunk 6]
```diff
@@ -6,12 +6,16 @@ services:
       context: .
       dockerfile: Dockerfile-mysterium-node
     container_name: mysterium-node
+    cap_add:
+      - NET_ADMIN
```
Comment (ewrogers) [inline docker-compose.yml]: I haven't had luck with `--cap-add=NET_ADMIN`. I needed to use `--privileged` for IP forwarding to work.

File: Dockerfile-mysterium-node (hunk 1 above)
Comment (Goodsmileduck) [inline Dockerfile-mysterium-node]: build not wotking without it, don't know why. but that probably about ubuntu version

File: Dockerfile-mysterium-node (hunk 2 above)
Comment (Goodsmileduck) [inline Dockerfile-mysterium-node]: yes, you should, that just a example. there env in docker-compose.yml

File: docker-compose.yml (hunk 6 above)
Comment (Goodsmileduck) [inline docker-compose.yml]: privileged is not good approach, would recommend it for debug. it also didn't work for me if I create tun in RUN step on build, with entrypoint it works.

File: Dockerfile-mysterium-node (hunk 4 above)
Comment (Goodsmileduck) [inline Dockerfile-mysterium-node]: so you need add RUN step to create /var/run/mysterium-node and chown

File: Dockerfile-mysterium-node (hunk 1 above)
Comment (Goodsmileduck) [inline Dockerfile-mysterium-node]: I have checked it again, dpkg didn't install dependecies and give a error, so build will be failed.
Other way to use apt-get directly
`RUN apt-get update \
    && apt install -y wget\
    && wget https://github.com/MysteriumNetwork/node/releases/download/${VERSION}/mysterium-node_linux_amd64.deb \
    && apt-get install -y ./mysterium-node_linux_amd64.deb \
    && rm -rf /var/cache/apk/* \
    && rm mysterium-node_linux_amd64.deb`

File: entrypoint-node.sh (hunk 3 above)
Comment (Goodsmileduck) [inline entrypoint-node.sh]: I have tested it, it didn't works. Sorry, I know less about mknod. entrypoint way is not a hack, I would recommend it. Later you can preheat some other stuff in this script.

**Pull request merged**